dashboard2/settings/jptt.md

```diff
@@ -154,9 +154,13 @@ HTML:
 
 #### Skip
 
-Use this processing mode to mark child node not to be translated. Order is important, skip rules must come after the generic * rule. The exclamation mark in the end of the rule is mandatory, so please use the following syntax: `! skip`
+Use this processing mode to mark child node not to be translated. **Order is important!**
+If you use `*` or `**` to select child nodes to be translated, you can use the `! skip` processing mode to select a child element as non-translatable, but please note, that skip rules must come after the generic `*` rule. Skip is the lowest level mode, but you can increase the priority by adding `!` or `!{number}`
```

Comment:
While this is true, the `!` switch is not only for `skip` rules, technically. Any rule can be appended to increase its priority above other matching rules, no?

Perhaps the `!` could be a separate heading on its own, along with commenting lines?